packages/keystore/src/aes-ctr-node.ts

```diff
@@ -11,6 +11,7 @@ const ALGORITHM = 'aes-256-ctr';
  *
  * @returns Promise<Keystore> object
  */
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Or else, remove the `async` from the method.

Reply:
Removing was my first thought.  But taking into consideration what was discussed [here](https://github.com/FuelLabs/fuels-ts/issues/832#issuecomment-1475614618) in the issue, I am under the impression that this function may become (real) `async` in the future. I believe @luizstacio can give us more context.

About the disabled rule, it was just a matter of choice. Since adding an `await` where is not needed will make the TS shows the 3 dots warning
![Screenshot from 2023-03-21 15-16-13](https://user-images.githubusercontent.com/30977845/226704032-0a01c6f5-c2ab-4c3a-9112-e90d95358fd8.png)

But yeah, we could also leave it like that.